glean-core/android/src/main/java/mozilla/telemetry/glean/Dispatchers.kt

```diff
@@ -117,12 +116,28 @@ internal object Dispatchers {
             // situations. If a library or thread that runs before init happens to record
             // between when the queueInitialTasks is set to false and the taskQueue finishing
             // launching, then that task could be executed out of the queued order.
+            val that = this
```

Comment:
If this is a common idiom, ignore my comment... setting `this` to `that` seems confusing.  Maybe `dispatchersObject` instead?

Reply:
This was just needed because the context of `this` changes inside of the closure and doing this captures the correct reference to `this`.  I'll rename it to be more clear.